### ISSUE TITLE
What is the relationship between train.py and tune*.py?

### ISSUE THREAD

**Issue by loopyli** [KMnP/vpt#39]
Hi,in my understanding, train.py is training prompts and heads. I think this is a fine-tuning process, so what is the role of tune*. py? I have read the explanation in README.md"call this one for tuning learning rate and weight decay for a model with a specified transfer type", but I am still a bit confused. Is tune*.py necessary for every task?And are there any order between train.py and tune*.py?

**Comment by KMnP** [comment]
hi, we provide `tune*.py` so one could easily grid-search hyper-parameters, and change the range of hyper-parameters to different needs. 

You could certainly use `train.py` and use 1 set of hyper-parameters.
